config/crossSections.yaml

```
@@ -0,0 +1,619 @@
+ST_tW_antitop:
+  crossSec: 35.85
+  reference: https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#Single_top
+  unc: 35.85 +0.90 -0.90 +1.70 -1.70
+ST_tW_top:
+  crossSec: 35.85
+  reference: https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#Single_top
+  unc: 35.85 +0.90 -0.90 +1.70 -1.70
+ST_t-channel_top:
+  crossSec: 136.02
+  reference: HTT AN-19-109
+  unc:
+ST_t-channel_antitop:
+  crossSec: 80.95
+  reference: HTT AN-19-109
+  unc:
+EWKWMinus2Jets_WToLNu_M-50:
+  crossSec: 20.25
+  reference: HTT AN-19-109
+  unc:
+EWKWPlus2Jets_WToLNu_M-50:
+  crossSec: 25.62
+  reference: HTT AN-19-109
+  unc:
+EWKZ2Jets_ZToLL_M-50:
+  crossSec: 3.987
+  reference: HTT AN-19-109
+  unc:
+TTTo2L2Nu:
+  crossSec: 88.29
+  reference: HTT AN-19-109
+  unc:
+TTToSemiLeptonic:
+  crossSec: 365.35
+  reference: HTT AN-19-109
+  unc:
+TTToHadronic:
+  crossSec: 377.96
+  reference: HTT AN-19-109
+  unc:
+ttHTobb:
+  crossSec: 0.2953
+  reference: Higgs_XSBR_YR4_update
+  unc:
+ttHToTauTau:
+  crossSec: 0.031805
+  reference: Higgs_XSBR_YR4_update
+  unc:
+ttHToNonbbORtautau:
+  crossSec: 0.17996
+  reference: Higgs_XSBR_YR4_update
+  unc:
+ttHToNonbb:
+  crossSec: 0.212
+  reference: tH AN-19-111
+  unc:
+TTWJetsToLNu:
+  crossSec: 0.2043
+  reference: https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#TT_X
+  unc: 0.2043 +- 0.0020
+TTWJetsToQQ:
+  crossSec: 0.4062
+  reference: https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#TT_X
+  unc: 0.4062 +- 0.0021
+TTZToLLNuNu_M-10:
+  crossSec: 0.2529
+  reference: https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#TT_X
+  unc: 0.2529 +- 0.0004
+TTZToQQ:
+  crossSec: 0.5297
+  reference: https://twiki.cern.ch/twiki/bin/viewauth/CMS/SummaryTable1G25ns#TT_X
+  unc: 0.5297 +- 0.0008
+TTWW:
+  crossSec: 0.00698
+  reference: ttH AN-19-111
+  unc:
+TTWZ:
+  crossSec: 0.002441
+  reference: XSDB
```

be more precise e.g. 
```suggestion
  reference: XSDB TTWZ_TuneCP5_13TeV-madgraph-pythia8 (modifiedOn 2021-06-21 09:55:55)
```

---

ok